packages/cdk/docs/big-decimal.md

```diff
@@ -0,0 +1,30 @@
+#### `@angular-ru/cdk/big-decimal`
+
+-   `compare`
+
+```ts
+import { BigDecimal, BigDecimalType } from '@angular-ru/cdk/big-decimal';
+
+type ComparableRangeResult = 0 | 1 | -1;
+
+const MIN_LONG_VALUE: number = new BigDecimal('-9223372036854775808');
+const MAX_LONG_VALUE: number = new BigDecimal('9223372036854775807');
+
+@Component()
+export class AppComponent {
+    constructor() {
+        const isLong: boolean = this.isLong('777');
+    }
+
+    public isLong(val: string): boolean {
+        if (val.match(/[^\d\-]/g)) {
+            return false;
+        } else {
+            const decimal: BigDecimalType = new BigDecimal(val);
+            const comparedLeft: ComparableRangeResult = decimal.compareTo(MIN_LONG_VALUE);
+            const comparedRight: ComparableRangeResult = decimal.compareTo(MAX_LONG_VALUE);
+            return !((comparedLeft === 1 && comparedRight === 1) || (comparedLeft === -1 && comparedRight === -1));
+        }
+    }
```

Comment:
what about create util out of the box

```ts
import { isLong } from '@angular-ru/cdk/big-decimal';
```

Reply:
that will be the next step, main task is add validator for long format
big-decimal is just the beginning